Doc/library/sys.rst

```diff
@@ -518,24 +518,24 @@ always available.
    ==============================  ==============================================================================================================
    attribute                       flag
    ==============================  ==============================================================================================================
-   :const:`debug`                  :option:`-d`
-   :const:`inspect`                :option:`-i`
-   :const:`interactive`            :option:`-i`
-   :const:`isolated`               :option:`-I`
-   :const:`optimize`               :option:`-O` or :option:`-OO`
-   :const:`dont_write_bytecode`    :option:`-B`
-   :const:`no_user_site`           :option:`-s`
-   :const:`no_site`                :option:`-S`
-   :const:`ignore_environment`     :option:`-E`
-   :const:`verbose`                :option:`-v`
-   :const:`bytes_warning`          :option:`-b`
-   :const:`quiet`                  :option:`-q`
-   :const:`hash_randomization`     :option:`-R`
-   :const:`dev_mode`               :option:`-X dev <-X>` (:ref:`Python Development Mode <devmode>`)
-   :const:`utf8_mode`              :option:`-X utf8 <-X>`
-   :const:`safe_path`              :option:`-P`
-   :const:`int_max_str_digits`     :option:`-X int_max_str_digits <-X>` (:ref:`integer string conversion length limitation <int_max_str_digits>`)
-   :const:`warn_default_encoding`  :option:`-X warn_default_encoding <-X>`
+   :attr:`!debug`                  :option:`-d`
+   :attr:`!inspect`                :option:`-i`
+   :attr:`!interactive`            :option:`-i`
+   :attr:`!isolated`               :option:`-I`
+   :attr:`!optimize`               :option:`-O` or :option:`-OO`
+   :attr:`!dont_write_bytecode`    :option:`-B`
+   :attr:`!no_user_site`           :option:`-s`
+   :attr:`!no_site`                :option:`-S`
+   :attr:`!ignore_environment`     :option:`-E`
+   :attr:`!verbose`                :option:`-v`
+   :attr:`!bytes_warning`          :option:`-b`
+   :attr:`!quiet`                  :option:`-q`
+   :attr:`!hash_randomization`     :option:`-R`
+   :attr:`!dev_mode`               :option:`-X dev <-X>` (:ref:`Python Development Mode <devmode>`)
+   :attr:`!utf8_mode`              :option:`-X utf8 <-X>`
+   :attr:`!safe_path`              :option:`-P`
+   :attr:`!int_max_str_digits`     :option:`-X int_max_str_digits <-X>` (:ref:`integer string conversion length limitation <int_max_str_digits>`)
+   :attr:`!warn_default_encoding`  :option:`-X warn_default_encoding <-X>`
```

Comment:
Following on from Alex's excellent suggestion:

*(I think a later PR perhaps ought fill these in with some more detail, but for now this is just about markup)*

```suggestion
   .. attribute:: flags.debug

      :option:`-d`

   .. attribute:: flags.inspect

      :option:`-i`

   .. attribute:: flags.interactive

      :option:`-i`

   .. attribute:: flags.isolated

      :option:`-I`

   .. attribute:: flags.optimize

      :option:`-O` or :option:`-OO`

   .. attribute:: flags.dont_write_bytecode

      :option:`-B`

   .. attribute:: flags.no_user_site

      :option:`-s`

   .. attribute:: flags.no_site

      :option:`-S`

   .. attribute:: flags.ignore_environment

      :option:`-E`

   .. attribute:: flags.verbose

      :option:`-v`

   .. attribute:: flags.bytes_warning

      :option:`-b`

   .. attribute:: flags.quiet

      :option:`-q`

   .. attribute:: flags.hash_randomization

      :option:`-R`

   .. attribute:: flags.dev_mode

      :option:`-X dev <-X>` (:ref:`Python Development Mode <devmode>`)

   .. attribute:: flags.utf8_mode

      :option:`-X utf8 <-X>`

   .. attribute:: flags.safe_path

      :option:`-P`

   .. attribute:: flags.int_max_str_digits

      :option:`-X int_max_str_digits <-X>` (:ref:`integer string conversion length limitation <int_max_str_digits>`)

   .. attribute:: flags.warn_default_encoding

      :option:`-X warn_default_encoding <-X>`
```

Reply:
For this one specifically, I think the table format is better. When you're reading the documentation here, you'll just want to follow the links to each option. Having each enumerated as a separate attribute entry feels unnecessarily verbose.

Iff we added a little more description for each entry, I think this would be worth it, but that feels like something that could maybe be postponed to a followup PR

Reply:
What we could do is a list-table similar to #108107, so we're able to retain the canonical links to ``.. attribute:: flags.blah``?

Reply:
I did not know you could do that with a list-table -- that sounds ideal, yes!

Reply:
List-Table suggestion:
```suggestion
   .. list-table::
      * - .. attribute:: flags.debug
        - :option:`-d`

      * - .. attribute:: flags.inspect
        - :option:`-i`

      * - .. attribute:: flags.interactive
        - :option:`-i`

      * - .. attribute:: flags.isolated
        - :option:`-I`

      * - .. attribute:: flags.optimize
        - :option:`-O` or :option:`-OO`

      * - .. attribute:: flags.dont_write_bytecode
        - :option:`-B`

      * - .. attribute:: flags.no_user_site
        - :option:`-s`

      * - .. attribute:: flags.no_site
        - :option:`-S`

      * - .. attribute:: flags.ignore_environment
        - :option:`-E`

      * - .. attribute:: flags.verbose
        - :option:`-v`

      * - .. attribute:: flags.bytes_warning
        - :option:`-b`

      * - .. attribute:: flags.quiet
        - :option:`-q`

      * - .. attribute:: flags.hash_randomization
        - :option:`-R`

      * - .. attribute:: flags.dev_mode
        - :option:`-X dev <-X>` (:ref:`Python Development Mode <devmode>`)

      * - .. attribute:: flags.utf8_mode
        - :option:`-X utf8 <-X>`

      * - .. attribute:: flags.safe_path
        - :option:`-P`

      * - .. attribute:: flags.int_max_str_digits
        - :option:`-X int_max_str_digits <-X>` (:ref:`integer string conversion length limitation <int_max_str_digits>`)

      * - .. attribute:: flags.warn_default_encoding
        - :option:`-X warn_default_encoding <-X>`
```

Reply:
I agree with Alex that a table makes sense here. Yay to more `list-table` markups; they are fantastic.